e2e/alphabet-card-detail.test.ts

```diff
@@ -12,15 +12,19 @@ const multiTap = async (tappable: ITappableAsync, numberOfTaps: number): Promise
     }
 };
 
+require('dotenv').config({ path: '.production.env' });
+
+const { TARGET_ALPHABET_NAME, BASE_API_URL, BROKEN_API_URL, BROKEN_ALPHABET_NAME } = process.env;
+
 describe('Detail', () => {
     beforeEach(async () => {
         await device.launchApp({
             launchArgs: {
                 configOverrides: {
                     config: {
                         env: {
-                            TARGET_ALPHABET_NAME: `alphabet-english`,
-                            BASE_API_URL: `http://10.0.2.2:3131/api`,
```

Comment:
If this already falls back to what is in the `.env`, then you really just need to remove the `BASE_API_URL` property altogether and we can set whatever we want here in the CI.

Reply:
We have addressed this now.